commands/dkcheck.ml

```diff
@@ -3,14 +3,9 @@ open Basic
 open Parser
 open Entry
 
-let verbose = ref false
-
 let eprint lc fmt =
-  if !verbose then
-    let (l,c) = of_loc lc in
-    Format.eprintf "line:%i column:%i " l c;
-    Format.kfprintf (fun _ -> prerr_newline ()) Format.err_formatter fmt
-  else Format.ifprintf Format.err_formatter fmt
```

Comment:
Le cas où le mode verbose n'est pas activé semble avoir disparu dans ta PR.

Reply:
Quand le paramètre de debug est < 1, ce texte ne s'affiche pas.

Reply:
The last line (`else Format.ifprintf Format.err_formatter fmt`) ignores the rest of the arguments. It was only required for the old function to have the right type. Now the ignoring is done inside the `debug`function.

Reply:
Ok, I should have read the documentation of Format.ifprintf before review your PR.